ui/lib/apps/SlowQuery/components/TimeRangeSelector.tsx

```diff
@@ -21,9 +21,10 @@ export interface TimeRange {
 
 export const DEF_TIME_RANGE: TimeRange = {
   recent: 30,
-  begin_time: 0,
-  end_time: 0,
+  begin_time: dayjs().unix() - 30 * 60,
+  end_time: dayjs().unix(),
 }
```

Comment:
yep, this part logic has some problem, I will try to figure it out after this PR is merged.